interface/reports/adjustments.php

```diff
@@ -0,0 +1,264 @@
+ *
+ * Copyright (C) 2017-2018 Terry Hill <teryhill@librehealth.io>
+ *
+ * LICENSE: This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0
+ * See the Mozilla Public License for more details.
+ * If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * @package LibreHealth EHR
+ * @author Terry Hill <teryhill@librehealth.io>
+ * @link http://librehealth.io
+ */
+
+ require_once("../globals.php");
+ require_once "$srcdir/patient.inc";
+ require_once "$srcdir/acl.inc";
+ require_once "$srcdir/options.inc.php";
+ require_once "$srcdir/formatting.inc.php";
+ $DateFormat = DateFormatRead();
+ $DateLocale = getLocaleCodeForDisplayLanguage($GLOBALS['language_default']);
+
+
+ $from_date = fixDate($_POST['form_from_date']);
+ $to_date   = fixDate($_POST['form_to_date'], date('Y-m-d'));
+
+
+if ($_POST['form_csvexport']) {
+  header("Pragma: public");
+  header("Expires: 0");
+  header("Cache-Control: must-revalidate, post-check=0, pre-check=0");
+  header("Content-Type: application/force-download");
+  header("Content-Disposition: attachment; filename=adjustment_codes.csv");
+  header("Content-Description: File Transfer");
+  // CSV headers:
+  if (true) {
+    echo '"Adjustment Code",';
+    echo '"Total",'. "\n";
+  }
+}
+else {
+?>
+<html>
+<head>
+<title><?php echo xlt('Adjustments'); ?></title>
+
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/overlib_mini.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery-1.9.1.min.js"></script>
+
+<script language="JavaScript">
+
+ $(document).ready(function() {
+  var win = top.printLogSetup ? top : opener.top;
+  win.printLogSetup(document.getElementById('printbutton'));
+ });
+
+</script>
+
+<link rel=stylesheet href="<?php echo $css_header;?>" type="text/css">
+
+<style type="text/css">
+
+/* specifically include & exclude from printing */
+@media print {
+    #report_parameters {
+        visibility: hidden;
+        display: none;
+    }
+    #report_parameters_daterange {
+        visibility: visible;
+        display: inline;
+    }
+    #report_results table {
+       margin-top: 0px;
+    }
+}
+
+/* specifically exclude some from the screen */
+@media screen {
+    #report_parameters_daterange {
+        visibility: hidden;
+        display: none;
+    }
+}
+
+</style>
+</head>
+
+<body class="body_top">
+
+<div id="overDiv" style="position:absolute; visibility:hidden; z-index:1000;"></div>
+
+<span class='title'><?php echo xlt('Report'); ?> - <?php echo xlt('Adjustments'); ?></span>
+
+<div id="report_parameters_daterange">
+    <?php date("d F Y", strtotime(oeFormatDateForPrintReport($_POST['form_from_date'])))
+    . " &nbsp; to &nbsp; ". date("d F Y", strtotime(oeFormatDateForPrintReport($_POST['form_to_date']))); ?>
```

Comment:
Does this date format function allow for localization?  If not, then Y m d is preferred.  If using Portuguese as the language doesn't result in "Januario" then we are just making trouble.

Reply:
It is the same date function in all the other reports so I hope so 8^)